src/Client.sol

```diff
@@ -6,36 +6,35 @@ import {DataCapTypes} from "filecoin-solidity/contracts/v0.8/types/DataCapTypes.
 import {DataCapAPI} from "filecoin-solidity/contracts/v0.8/DataCapAPI.sol";
 import {Errors} from "./libs/Errors.sol";
 import {CBORDecoder} from "filecoin-solidity/contracts/v0.8/utils/CborDecode.sol";
-import {IAllocation} from "./interfaces/IAllocation.sol";
+import {IClient} from "./interfaces/IClient.sol";
 
-contract Allocation is Initializable, IAllocation {
+contract Client is Initializable, IClient {
```

Comment:
Add https://docs.openzeppelin.com/contracts/5.x/utilities#multicall so that FE will be able to grant allowance and set SPs using one transaction.